src/xunit.performance.api/ETWProfiler.cs

```diff
@@ -107,9 +116,142 @@ public static void Profile(string assemblyFileName, string sessionName, Action a
 
             TraceEventSession.MergeInPlace(userFullFileName, Console.Out);
             WriteInfoLine($"ETW Tracing Session saved to \"{userFullFileName}\"");
+            collectOutputFilesCallback(userFullFileName);
+
+            var assemblyModel = GetAssemblyModel(assemblyFileName, userFullFileName, sessionName, performanceTestMessages);
```

Comment:
This is not really a Xunit specific construct (more like a copied schema/model of its output). This is a type used to serialized collected data into something that BenchView tools already parse. In addition, we would not break backward compatibility by switching to the new API (without creating a new format on the BenchView tools to parse). I am opened to redefine this later on if needed.